unit/src/main/java/org/apache/falcon/unit/LocalExtensionManager.java

```diff
@@ -88,7 +104,19 @@ public APIResult submitAndSchedulableExtensionJob(String extensionName, String j
         return new APIResult(APIResult.Status.SUCCEEDED, "Extension job submitted successfully" + jobName);
```

Comment:
Not related to this change. But I updated the return message for submitAndScheduleExtensionJob Api